tests/CoreBundle/Controller/CohortControllerTest.php

```diff
@@ -85,180 +85,6 @@ public function testGetAllCohorts()
     /**
      * @group controllers_a
      */
-    public function testPostCohort()
```

Comment:
ideally, i'd remove that endpoint right now and here.
perhaps we can hollow it out and have it provide no functionality at all, or have it emit a warning.
let's discuss tomorrow.